From d87ff2db77ed6d246fca80acb825272e8c7baa78 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:33 +0800
Subject: [PATCH 49/60] Revert "drm/nouveau: fix another off-by-one in
 nvbios_addr"

This reverts commit ca0742a8ed547de5c14100f3ffab8540159be899.
---
 drivers/gpu/drm/nouveau/nvkm/subdev/bios/base.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/nouveau/nvkm/subdev/bios/base.c b/drivers/gpu/drm/nouveau/nvkm/subdev/bios/base.c
index f0368d9a0154d..8bff14ae16b0e 100644
--- a/drivers/gpu/drm/nouveau/nvkm/subdev/bios/base.c
+++ b/drivers/gpu/drm/nouveau/nvkm/subdev/bios/base.c
@@ -33,7 +33,7 @@ nvbios_addr(struct nvkm_bios *bios, u32 *addr, u8 size)
 {
 	u32 p = *addr;
 
-	if (*addr >= bios->image0_size && bios->imaged_addr) {
+	if (*addr > bios->image0_size && bios->imaged_addr) {
 		*addr -= bios->image0_size;
 		*addr += bios->imaged_addr;
 	}
-- 
2.25.1

